stubs/docker/docker/utils/utils.pyi

```diff
@@ -56,8 +56,8 @@ def parse_host(
 ) -> Literal["http+unix:///var/run/docker.sock"]: ...
 @overload
 def parse_host(addr: str | None, is_win32: bool = False, tls: bool = False) -> str | bytes: ...
-def parse_devices(devices: Iterable[str | dict[str, Incomplete]]) -> list[dict[str, Incomplete]]: ...
-def kwargs_from_env(environment: Mapping[str, Incomplete] | None = None) -> _EnvKWArgs: ...
+def parse_devices(devices: Iterable[str | dict[str, str]]) -> list[dict[str, str]]: ...
+def kwargs_from_env(environment: Mapping[str, AnyStr] | None = None) -> _EnvKWArgs: ...
```

Comment:
`AnyStr` is a type var and as such needs a second use. You probably just want to use `str | bytes` here.